### PR TITLE
changes to parse altText fixes #154

### LINKED ISSUE #154: Request for AltText for StaticResource companion ad
Hello and thanks for this great tool!

We noticed that the parser doesn't parse altText attribute if a staticresource is being served
in file vast-client-js/src/companionad.coffee the VASTCompanionAd class doen't define an alttext property
``` javascript
class VASTCompanionAd
    constructor: ->
        @id = null
        @width = 0
        @height = 0
        @type = null
        @staticResource = null
        @htmlResource = null
        @iframeResource = null
        @companionClickThroughURLTemplate = null
        @trackingEvents = {}

module.exports = VASTCompanionAd
```
Is this a bug? or is there any other way we can fetch the alttext property?
Is this going to be developed?

thanks a lot in advance

### PR DESCRIPTION
**Pull request by vagelkara**
This fixes #154. Have a look and tell me if all its ok!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 dist/*
 node_modules/*
 bower_components/*
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+vast-client.js
```
Comment (michaelvial) [inline .gitignore]: Don't gitignore `vast-client.js`, we need to commit it at every release

File: test/parser.coffee [hunk 2]
```diff
@@ -218,6 +218,13 @@ describe 'VASTParser', ->
 
                         it 'should have 1 url for creativeView event', =>
                             companion.trackingEvents['creativeView'].should.eql ['http://example.com/companion1-creativeview']
+				
+
```
Comment (michaelvial) [inline test/parser.coffee]: One blank line between test is enough

File: test/sample.xml [hunk 3]
```diff
@@ -53,13 +53,16 @@
...
+				<![CDATA[Sample Alt Text Content!!!!]]>
+			  </AltText>
+			  <TrackingEvents>
                 <Tracking event="creativeView"><![CDATA[http://example.com/companion1-creativeview]]></Tracking>
               </TrackingEvents>
               <CompanionClickThrough><![CDATA[http://example.com/companion1-clickthrough]]></CompanionClickThrough>
               <CompanionClickTracking><![CDATA[http://example.com/companion1-clicktracking-first]]></CompanionClickTracking>
               <CompanionClickTracking><![CDATA[http://example.com/companion1-clicktracking-second]]></CompanionClickTracking>
-            </Companion>
+			  </Companion>
```
Comment (michaelvial) [inline test/sample.xml]: XML indentation problem here.

**Comment by vagelkara** [comment]
Just changed 3 files, its my first pull request in a public project, so hope to help!!
@michaelvial Have a look!!

**Comment by michaelvial** [comment]
👍 @vagelkara

**Pull request merged**